packages/runtime/package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.0.1",
   "type": "commonjs",
   "devDependencies": {
+    "@chakra-ui/react": "^2.5.1",
```

Comment:
What runtime need chakra-ui @arrizalamin ?

Reply:
this was from formula-editor before refactored